lib/stripe/callbacks.rb

```diff
@@ -65,12 +74,16 @@ module Callbacks
     callback 'recipient.created'
     callback 'recipient.deleted'
     callback 'recipient.updated'
+    callback 'review.closed'
+    callback 'review.opened'
+    callback 'scheduled_query_run.created'
     callback 'sku.created'
     callback 'sku.deleted'
     callback 'sku.updated'
     callback 'source.canceled'
     callback 'source.chargeable'
     callback 'source.failed'
+    callback 'source.transaction.created'
     callback 'transfer.created'
     callback 'transfer.failed'
     callback 'transfer.paid'
```

Comment:
also it looks like 'transfer.failed' and 'transfer.paid' have been removed from the docs. You might want to remove those too. Thanks!

Reply:
also, there's now `transfer.reversed` and `transfer.updated`